larva.config.js

```diff
@@ -8,15 +8,20 @@ module.exports = {
 		testBaseUrl: 'http://localhost:' + LARVA_PORT + '/larva/__tests__/',
 		testScenario: {
 			delay: 1000,
-			misMatchThreshold: 0.5,
+			misMatchThreshold: 5,
 		},
 		testPaths: [ 'profile', 'profile-index', 'vlanding', 'css', 'hub' ],
 		backstopConfig: {
 			engineOptions: {
+				headless: true,
 				args: [
 					'--no-sandbox',
 					'--proxy-server=127.0.0.1:' + LARVA_PORT,
 					'--proxy-bypass-list=<-loopback>',
+					'--ignore-certificate-errors',
+					'--disable-setuid-sandbox',
+					'--disable-accelerated-2d-canvas',
+					'--disable-gpu'
```

Comment:
Oddly, the tests pass even though there's a complaint about the missing comma. I think we should merge this and open an issue for further investigation.